src/IIIFPresentation/API/Helpers/BatchHelper.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Models.Database.Collections;
+using Models.Database.General;
+using Repository;
+
+namespace API.Helpers;
+
+public static class BatchHelper
+{
+    public static async Task SaveBatchesInDlcs(this List<DLCS.Models.Batch> batches, 
```

Comment:
Name of this method makes it sound like it's saving _in_ the DLCS rather than saving in the DB